aiohttp/typedefs.py

```diff
@@ -17,8 +17,23 @@
 
 Query = _Query
 
-DEFAULT_JSON_ENCODER = json.dumps
-DEFAULT_JSON_DECODER = json.loads
+# Try to use orjson for better performance, fallback to standard json
+try:
+    import orjson
+
+    def _orjson_dumps(obj: Any) -> str:
+        """orjson encoder that returns str (like json.dumps)."""
+        return orjson.dumps(obj).decode("utf-8")
```

Comment:
As mentioned (https://github.com/aio-libs/aiohttp/issues/10795#issuecomment-2830793594) we probably want a new method first which takes bytes directly. Otherwise, we're not going to be looking at any performance improvement after we've converted to str and then back to bytes.